scripts/imgtool/keys/general.py

```diff
@@ -30,6 +32,25 @@ def emit_c_public(self, file=sys.stdout):
                 len_format="const unsigned int {}_pub_key_len = {{}};".format(self.shortname()),
                 file=file)
 
+    def emit_raw_public(self, file=sys.stdout):
+        sys.stdout.close = lambda: None
+        with (open(file, 'wb') if file and file is not sys.stdout else sys.stdout.buffer) as file:
+            file.write(self.get_public_bytes())
+
+    def emit_c_public_hash(self, file=sys.stdout):
```

Comment:
Please move it below the `emit_c_public()` implementation.

Reply:
Done